sharding/sharding.go

```diff
@@ -62,12 +62,6 @@ func NewFerry(config *Config) (*ShardingFerry, error) {
 
 	logger := logrus.WithField("tag", "sharding")
 
-	ferry.ErrorHandler = &ghostferry.PanicErrorHandler{
-		Ferry:             ferry,
-		ErrorCallback:     config.ErrorCallback,
-		DumpStateToStdout: false,
```

Comment:
Removing this means that now sharding will dump state to stdout, as that's a default value inside Ghostferry. We should either preserve the previous behaviour, or make this configurable.